windows/Dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM golang:1.19-nanoserver AS workspace
+LABEL builder=true
+
+ENV CGO_ENABLED=0
+#ENV GO111MODULE=on
```

Comment:
```suggestion
```
Do we need this?

Reply:
👍 I've removed the line during rebase/cleanup.